Vagrantfile

```diff
@@ -1,27 +0,0 @@
-Vagrant.configure("2") do |config|
-    if Vagrant.has_plugin?("vagrant-vbguest")
-        config.vbguest.auto_update = false
-    end
-
-    config.vm.box = "fedora/30-cloud-base"
```

Comment:
Considering that this file still has F30 hardcoded, we probably don't need it :sweat_smile: